src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -337,26 +322,27 @@ namespace VentilatedSlab {
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames.allocate(NumNumbers);
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames = cNumericFields;
             UtilityRoutines::IsNameEmpty(state, state.dataIPShortCut->cAlphaArgs(1), CurrentModuleObject, ErrorsFound);
+            auto & ventSlab = state.dataVentilatedSlab->VentSlab(Item);
 
-            state.dataVentilatedSlab->VentSlab(Item).Name = state.dataIPShortCut->cAlphaArgs(1);
-            state.dataVentilatedSlab->VentSlab(Item).SchedName = state.dataIPShortCut->cAlphaArgs(2);
+            ventSlab.Name = state.dataIPShortCut->cAlphaArgs(1);
+            ventSlab.SchedName = state.dataIPShortCut->cAlphaArgs(2);
             if (lAlphaBlanks(2)) {
-                state.dataVentilatedSlab->VentSlab(Item).SchedPtr = DataGlobalConstants::ScheduleAlwaysOn;
+                ventSlab.SchedPtr = DataGlobalConstants::ScheduleAlwaysOn;
             } else {
-                state.dataVentilatedSlab->VentSlab(Item).SchedPtr =
+                ventSlab.SchedPtr =
                     GetScheduleIndex(state, state.dataIPShortCut->cAlphaArgs(2)); // convert schedule name to pointer
-                if (state.dataVentilatedSlab->VentSlab(Item).SchedPtr == 0) {
+                if (ventSlab.SchedPtr == 0) {
                     ShowSevereError(state,
                                     CurrentModuleObject + "=\"" + state.dataIPShortCut->cAlphaArgs(1) + "\" invalid " + cAlphaFields(2) + "=\"" +
```

Comment:
Thing 3. If we've already assigned `state.dataIPShortCut->cAlphaArgs(1)` to `ventSlab.Name`, we should use ventSlab.Name since it is more readable. (also `format` instead of string concatenation).

Reply:
Is it preferable to use `format` whenever possible (even when there is no `string_view` involved)?

Reply:
Yes, it's supposedly [significantly faster](https://github.com/fmtlib/fmt) and smaller than `libc++` and even`vsprintf`. 

I'm not 100% sure about this, but I think each `std::string::operator+` does its own memory re-allocation so if you have a chain of five of them you end up doing five reallocations instead of one. This, by the way, is why you have to construct a `string` out of a `string_view` when you apply `operator+` to it, because a `string_view` does not own memory than  you can re-allocate.

Reply:
Got it. Thank you for explaining.